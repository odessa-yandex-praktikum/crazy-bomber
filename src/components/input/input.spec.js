import React from 'react';
import renderer from 'react-test-renderer';
import {Input} from './index';

it('renders correctly', () => {
    const tree = renderer.create(<Input nameField="" textError="" />).toJSON();
    expect(tree).toMatchSnapshot();
});

it('renders correctly with InputHTMLAttributes', () => {
    const tree = renderer
        .create(<Input nameField="name" textError="error name" value="123" border="2" />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
