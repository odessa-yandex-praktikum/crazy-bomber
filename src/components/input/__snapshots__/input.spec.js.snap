// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container__input-with-error"
>
  <div
    className="container__input"
  >
    <span>
      
    </span>
    <input
      className="input"
    />
  </div>
  <span
    className="input__error-text"
  >
    
  </span>
</div>
`;

exports[`renders correctly with InputHTMLAttributes 1`] = `
<div
  className="container__input-with-error"
>
  <div
    className="container__input"
  >
    <span>
      name
    </span>
    <input
      border="2"
      className="input"
      value="123"
    />
  </div>
  <span
    className="input__error-text"
  >
    error name
  </span>
</div>
`;
